hrjobcontract/tests/phpunit/CRM/Hrjobcontract/BAO/HRJobContractTest.php

```diff
@@ -170,6 +170,225 @@ public function testGetActiveContractsWithDetailsCanReturnOnlyContractsOfASpecif
     $this->assertNull($contact2ActiveContracts[0]['period_end_date']);
   }
 
+  public function testGetActiveContractWithDetailsShouldReturnTheDetailsFromTheLatestDetailsRevision() {
+    $this->createContacts(1);
```

Comment:
Yes, but this branch doesn't have the recent updates on staging yet. Once I update it (not on this pr) I switch things to use Fabricators